EIC-EA/EIC-community-D8
# Feature/eicnet forms style

### What I did

- adapt style off all content


### How to test

- go to /admin/modules
- active _EIC Dummy Content Type_
- add content dummy and check the form is ok
